examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/OverviewViewModel.kt

```diff
@@ -103,9 +106,41 @@ class OverviewViewModel(private val interactionHandler: OverviewInteractionHandl
         interactionHandler.syncAttributes()
     }
 
+    fun onFetchVCsClicked() {
+        viewModelScope.launch {
+            val virtualCurrencies: VirtualCurrencies = Purchases.sharedInstance.awaitGetVirtualCurrencies()
```

Comment:
Are we planning on using this for something other than logs? Could be good to add a section to visualize the virtual currency to the purchase tester app I think.